.circleci/config.yml

```diff
@@ -168,8 +170,304 @@ jobs:
           command: |
             echo "Dummy job."
 
+  go-lint-test:
+    parameters:
+      working_directory:
+        description: Working directory
+        type: string
+    docker:
+      - image: mslipper/go-builder:latest
+      - image: cimg/postgres:14.1
+    steps:
+      - go-lint-test:
+          working_directory: <<parameters.working_directory>>
+
+  go-lint-test-build:
+    parameters:
+      binary_name:
+        description: Binary name to build
+        type: string
+      working_directory:
+        description: Working directory
+        type: string
+    docker:
+      - image: mslipper/go-builder:latest
+      - image: cimg/postgres:14.1
+    steps:
+      - go-lint-test:
+          working_directory: <<parameters.working_directory>>
+      - run:
+          name: Build
+          command: make <<parameters.binary_name>>
+          working_directory: <<parameters.working_directory>>
+
+  yarn-monorepo:
+    docker:
+      - image: mslipper/slither:latest
+    steps:
+      - restore_cache:
+          keys:
+            - v1-source-{{ .Branch }}-{{ .Revision }}
+            - v1-source-{{ .Branch }}
+      - checkout
+      - save_cache:
+          key: v1-source-{{ .Branch }}-{{ .Revision }}
+          paths:
+            - ".git"
+      - restore_cache:
+          keys:
+            - v1-yarn-install-{{ checksum "yarn.lock" }}
+            - v1-yarn-install
+      - run:
+          name: Install dependencies
+          command: yarn --frozen-lockfile
+      - save_cache:
+          key: v1-yarn-install-{{ checksum "yarn.lock" }}
+          paths: 
+            - node_modules
+            - packages/common-ts/node_modules
+            - packages/contracts/node_modules
+            - packages/core-utils/node_modules
+            - packages/data-transport-layer/node_modules
+            - packages/message-relayer/node_modules
+            - packages/replica-healthcheck/node_modules
+            - packages/sdk/node_modules
+            - integration-tests/node_modules
+      - run:
+          name: Build monorepo
+          command: yarn build
+      - save_cache:
+          key: v1-yarn-build-{{ .Revision }}
+          paths:
+            - "."
+
+  contracts-slither:
+    docker:
+      - image: mslipper/slither:latest
```

Comment:
Same question as above, would this be moved to `ethereum-optimism/slither`?

Reply:
Yep, will do this tomorrow.